server/api/signup.js

```diff
@@ -142,7 +142,12 @@ internals.applyRoutes = function (server, next) {
                 }],
                 session: ['linkUser', 'linkAccount', function (results, done) {
 
-                    Session.create(results.user._id.toString(), done);
+                    const userAgent = Useragent.lookup(request.headers['user-agent']);
+                    const ip = request.info.remoteAddress;
```

Comment:
In the case of environments like Heroku, and others that are fronted by a load balancer, we'd want to check that first.

Ex: `const ip = request.headers['x-forwarded-for'] || request.info.remoteAddress;`